polkadot/scripts/update-chain-spec-with-fast-rococo.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash -eu
```

Comment:
@PierreBesson @BulatSaif 
Will this script be useful for you?
Are you going to directly use it, or could it be removed?

Also question about pre-built rococo runtime delivered in docker image. Would that be useful for you?

Reply:
Yes this script is indeed very useful. Thank you. However it doesn't have to be in this repo as it is quite a special use case. I think it would be enough to document the `ROCOCO_EPOCH_DURATION` variable in a readme file.

Reply:
removed in: [4325737](https://github.com/paritytech/polkadot-sdk/pull/1490/commits/43257373908a53c1755a69f82410830235c36807)